apps/README.md

```diff
@@ -0,0 +1,26 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
+
+## Launch `FluentUI Tester` app on iOS
+
+Prereq: FluentUI Tester on iOS can only run on a Mac.
+
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire Fluent UI React Native repository. E.g. from the root of the repo:
+```
+    yarn && yarn build
+```
+
+2. Then go into `apps/ios/src` folder and run pod install:
+
+```
+    cd apps/ios/src
+    pod install
+```
+Note: if you get the error: "CocoaPods could not find compatible versions for pod "MicrosoftFluentUI"," you will need to run `pod repo update` and then run `pod install` again.
```

Comment:
You can simplify this some what by just running `pod install --repo-update`

Reply:
Thanks, fixed in new commit.